base/linalg/generic.jl

```diff
@@ -387,9 +387,9 @@ function generic_vecnormp(x, p)
     else
         T = typeof(float(norm(v)))
     end
-    spp::promote_type(Float64, T) = p
+    spp::promote_type(Float64,T) = p
```

Comment:
Hm, given `promote_type(Float64, T)` is a function call, retaining the space may be more consistent with style elsewhere? (Likewise a couple lines below.)